docs/cars.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+import jinja2
+import os
+from collections import defaultdict
+from sortedcontainers import SortedList
+
+from common.basedir import BASEDIR
+from common.params import Params
+from selfdrive.car.car_helpers import interfaces, get_interface_attr
+from selfdrive.car.hyundai.radar_interface import RADAR_START_ADDR as HKG_RADAR_START_ADDR
+from selfdrive.test.test_routes import non_tested_cars
+from selfdrive.car.docs import Tier, Column, StarColumns, Footnote, MAKES_GOOD_STEERING_TORQUE, get_footnote, get_star_icon
+
+
+class Car:
+  def __init__(self, car_info, CP):
+    self.make, self.model = car_info.name.split(' ', 1)
+    self.row, star_count = self.get_row(car_info, CP)
+    self.tier = {5: Tier.GOLD, 4: Tier.SILVER}.get(star_count, Tier.BRONZE)
+
+  def get_row(self, car_info, CP):
+    # TODO: add YouTube videos
+    # Some minimum steering speeds are not yet in CarParams
+    min_steer_speed = CP.minSteerSpeed
+    if car_info.min_steer_speed is not None:
+      min_steer_speed = car_info.min_steer_speed
+      assert CP.minSteerSpeed == 0, "Minimum steer speed set in both CarInfo and CarParams for {}".format(CP.carFingerprint)
+
+    min_enable_speed = CP.minEnableSpeed
```

Comment:
Prius V specifies stop_and_go=True, so `minEnableSpeed` is -1 when you only get that with a sDSU. Perhaps we need to fix `stop_and_go` based on sDSU detection